takwimu/templates/profile/profile_detail_takwimu.html

```diff
@@ -42,7 +42,7 @@ <h3 class="chart-header">Population</h3>
           <h3 class="chart-header">Population</h3>
           {% include 'profile/_blocks/_stat_list.html' with stat=demographics.total_population stat_type='number' %}
         </div>
-        <div class="column-three-quarters" id="chart-histogram-demographics-residence_dist" data-stat-type="scaled-percentage" data-chart-title="Population by residence" data-initial-sort="-value"></div>
+        <div class="column-three-quarters" id="chart-column-demographics-residence_dist" data-stat-type="scaled-percentage" data-chart-title="Population by residence" data-initial-sort="-value"></div>
```

Comment:
Some embedded charts will still use `histogram` instead of `column`. Please confirm that there is a logic in the embed script to convert from `histogram` to `column` if we're going to support `column` only i.e. I'm assuming that's why you're changing `histogram` to `column`